git_remote_dropbox/__init__.py

```diff
@@ -692,9 +692,10 @@ def __getitem__(self, key):
 def main():
     # configure system
     stdout_to_binary()
-
     name, url = sys.argv[1:3]
-    url = url.lower()
+    url_parse = urlparse(url.lower())
+    token_key = url_parse.username or 'token'
+    url = url_parse.scheme + '://' + url_parse.hostname + url_parse.path
```

Comment:
This seems like a hacky way of doing things - why not just use URL parsing for the whole thing, then? Why reassemble this and then use string parsing?